test/bench/document_test.ts

```diff
@@ -521,6 +603,54 @@ const tests = [
       benchmarkObject(10000);
     },
   },
+  {
+    name: 'Document#tree 100',
+    run: (): void => {
+      benchmarkTree(100);
+    },
+  },
+  {
+    name: 'Document#tree 1000',
+    run: (): void => {
+      benchmarkTree(1000);
+    },
+  },
+  {
+    name: 'Document#tree 10000',
+    run: (): void => {
+      benchmarkTree(10000);
+    },
+  },
```

Comment:
```suggestion
```

Reply:
To keep this benchmark, the total runtime in CI process is currently quite slow. Therefore, it would be better to remove this benchmark. I hope we can uncomment this benchmark after improving the performance of the Tree later.